app/views/admin/loans/timeline/_edit_project_step.html.slim

```diff
@@ -0,0 +1,43 @@
+div.panel-body
+
+  = simple_form_for([:admin, step],
+          remote: true,
+          html: { class: 'project-step-form' },
+          defaults: {html5: true}) do |f|
+
+    .error_notification
+      p.alert.alert-danger
+        button.close type="button" data-hide="alert" x
+        i.fa.fa-exclamation-circle>
+        = I18n.t(:error_notification)
+
+    = f.error_notification
+    input type="hidden" id="deleted_locales" name="project_step[deleted_locales]" value="[]"
+
+    .step_form_left.col-sm-8
+      - step.used_locales.each do |l|
+        .step-form-left-group.language-block data-locale="#{l}"
+          .clearfix
+            = f.input "locale_#{l}".to_sym, label: false, collection: locale_options(step.permitted_locales), wrapper: :step_form_left_2, input_html: {class: 'locale'}, include_blank: false
+            = f.input "summary_#{l}".to_sym, wrapper: :step_form_left_12, input_html: {class: 'summary'}, placeholder: I18n.t(:summary, locale: l)
+          = f.input "details_#{l}".to_sym, as: :text, wrapper: :step_form_left_12, input_html: {class: 'details', rows: 3}, placeholder: I18n.t(:details, locale: l)
+          a.remove_language id="remove_#{l}_language" href="#" data-locale="#{l}" Remove
+          .clearfix
+      a.add_language id="add_another_language" href="#" data-available-locales="#{step.unused_locales.as_json}" Add another language
+
+    .step_form_right.col-sm-4
+      .clearfix
+        = f.input :step_type_value, label: 'Type', collection: ProjectStep.step_type_option_set.translated_list,
+                wrapper: :step_form_right_9, include_blank: '---', input_html: {class: 'type'}
+      = f.input :scheduled_date, label: 'Scheduled', as: :date, wrapper: :step_form_right_9, input_html: {class: 'date'}
+      = f.input :completed_date, label: 'Completed', as: :date, wrapper: :step_form_right_9, input_html: {class: 'date'}
+      .form-group.date.optional.project_step_completed_date
+        = label_tag(:is_finalized, "Finalized", class: "date optional col-sm-3 checkbox-label")
+        .col-sm-9
+          = f.check_box(:is_finalized, class: "step_form_right_9 finalized")
```

Comment:
Please be consistent in whether to use parens or not for method calls in slim. I'm guessing the norm is no parens.